<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HotkeyInput</class>
 <widget class="QDialog" name="HotkeyInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hotkey Register</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>120</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Modifiers</string>
   </property>
   <widget class="QCheckBox" name="hkmod_alt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>ALT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hkmod_shift">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>SHIFT</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hkmod_ctrl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>90</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>CTRL</string>
    </property>
   </widget>
  </widget>
  <widget class="QPlainTextEdit" name="text_keys">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No Hotkey Found for this Button. &lt;/span&gt;&lt;br/&gt;Register one to save it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Keys</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HotkeyInput</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HotkeyInput</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
